db/migrate/20200601091307_fix_fields_on_person_again.rb

```diff
@@ -0,0 +1,6 @@
+class FixFieldsOnPersonAgain < ActiveRecord::Migration[5.2]
+  def change
+    remove_column :profiles, :last_synced_with_nomis,  :datetime
+    add_column    :people,   :latest_nomis_booking_id, :datetime
```

Comment:
I don't think `latest_nomis_booking_id` is a `datetime` ?

Reply:
ah yes it should be `integer`, also are we not removing `latest_nomis_booking_id ` from profiles (or maybe not yet)?

Reply:
That's coming later @cwrw